vmcnet/train/default_config.py

```diff
@@ -238,18 +246,22 @@ def get_default_model_config() -> Dict:
                     "backflow": {
                         "ndense_list": ((256, 16), (256, 16), (256, 16), (256,)),
                         "kernel_init_unmixed": orthogonal_init,
+                        "kernel_init_transformer": orthogonal_init,
```

Comment:
Out of curiosity, do these initializers make sense for transformers? I don't know anything about it but just wondering if there are known initializers that work well for transformers and if these are sensible based on that.

Reply:
i can take a look on this later, here ionclude this just to pass the test